Example/Tests/Transaction/EthTransactionIT.swift

```diff
@@ -14,14 +14,19 @@ import Quick
 @testable import Web3Swift
 
 final class EthTransactionIT: XCTestCase {
+    
+    let transactionHash = EthTransactionHash(
```

Comment:
Let's make `transactionHash` and `network` computed properties just in case the acquire any state in the future. That way test runs order is not going to influence the behavior of either.